spyder/config/base.py

```diff
@@ -508,15 +505,10 @@ def translate_dumb(x):
 
     import gettext
     try:
-        _trans = gettext.translation(modname, locale_path, codeset="utf-8")
-        lgettext = _trans.lgettext
+        _trans = gettext.translation(modname, locale_path)
 
         def translate_gettext(x):
-            y = lgettext(x)
-            if is_text_string(y):
-                return y
-            else:
-                return to_text_string(y, "utf-8")
+            return _trans.gettext(x)
         return translate_gettext
     except Exception:
```

Comment:
```suggestion
    except OSError:
```

Reply:
That's a good point; crashing at startup would be bad!
There is an alternative: have `OSError` fall back to the dumb non-translation function, and any other exception print a meaningful warning message to stderr and then continue with the dumb function. But this would have to use `print` directly (or something like `debug_print`), because logging is not yet enabled at that point (as I discovered while trying to debug!).

Reply:
Actually, even simpler: stick with `Exception` and print a debugging output in all cases.

Reply:
Ok, I like this suggestion better. Let's improve it a bit for readability:

```suggestion
    except Exception as exc:
        print(
            f"Could not load translation due to: "
            f"{exc.__class__.__name__} - {exc}",
            file=sys.stderr
        )

```

Reply:
Maybe "translations" rather than "translation", otherwise great.

Reply:
(I didn't use f-strings in my original version, as I'd forgotten they were already introduced in Python 3.6!)